### PR TITLE
Fixed bug #18342: Default ranking help text is missing helpm icon

### PR DESCRIPTION
**Pull request by gabrieljenik**
Icon added.

### CODE REVIEW

**Comment by Shnoulle** [comment]
The class is sr-only : no need icon ?

File: assets/packages/questions/ranking/scripts/ranking.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ var RankingQuestion = function (options) {
         rankingID = "javatbd" + rankingName;
 
     //define HTML snippets
-    var screenReader = "<div class='sr-only'>" + $('#question' + questionId + ' .em_default').html() + "</div><div aria-hidden='true'>" + LSvar.lang.rankhelp + "</div>"
+    var screenReader = "<div class='sr-only'>" + $('#question' + questionId + ' .em_default').html() + "</div><div aria-hidden='true'><span class='fa fa-exclamation-circle' aria-hidden='true'></span> " + LSvar.lang.rankhelp + "</div>"
```
Comment (Shnoulle) [inline assets/packages/questions/ranking/scripts/ranking.js:22]: 1. It seems to be the screenreader help, not the help for other user.
2. Maybe best is to copy current help and replace the text : then it came from twig file (template can replace this twig file)

Comment (gabrieljenik) [reply assets/packages/questions/ranking/scripts/ranking.js:22]: > 1. It seems to be the screenreader help, not the help for other user.

Well. Not sure about that. The icon appears after this.
image

> Maybe best is to copy current help and replace the text : then it came from twig file (template can replace this twig file)

The ticket is just about showing the icon. This is v5. Not sure it is worth here.

**Comment by gabrieljenik** [comment]
> 

It is like that on the twig files.

**Comment by Shnoulle** [comment]
The icon is here without JS

**Review by Shnoulle — APPROVED** [review]
Like to have it by twig, but don't find a good way.

**Pull request merged**